pom.xml

```diff
@@ -521,7 +526,7 @@
 						-->
 						<argLine>-Duser.language=en -Djna.nosys=true -XX:-UseSplitVerifier</argLine> <!-- only one argLine element is processed -->
 					</configuration>
-					<version>2.19.1</version>
+					<version>2.20</version>
```

Comment:
As i'm waiting for it, in my jnatest branch, since around 2 weeks already now, for re test JNA 4.3.0 issue, so i can tell that it will not work until few days.
I didn't saw it in mvnrepository as it's not yet realized/pushed yet, AFAIK.